libs/ai-embedded/set-env.mjs

```diff
@@ -0,0 +1,29 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { config } from 'dotenv';
+import path from 'node:path';
+
+config();
+
+const environmentProdFilePath = path.resolve('./libs/ai-embedded/src/environments/environment.prod.ts');
+
+const variableMap = {
+  appUrl: 'APP_URL',
+};
+
+const updateEnvironmentFile = (filePath) => {
+  let content = readFileSync(filePath, 'utf8');
+
+  Object.keys(variableMap).forEach(key => {
+    const envKey = variableMap[key];
+    const value = process.env[envKey];
+    if (value) {
+      const regex = new RegExp(`(${key}:\\s*).*(,)`, 'g');
+      content = content.replace(regex, `$1'${value}'$2`);
+    }
+  });
+
+  writeFileSync(filePath, content, 'utf8');
+  console.log(`Updated ${filePath}`);
```

Comment:
There is a chance for no updates to be made. Maybe we would like to inform the user about the update, only when there was one?

Reply:
```ts
  if (updatedContent !== content) {
    writeFileSync(filePath, updatedContent, 'utf8');
    console.log(`Updated ${filePath}`);
  } else {
    console.log(`No changes made to ${filePath}`);
  }
```

Reply:
Or without the logic included in the `else`

Reply:
It is a script for a prebuild process. Variables are always replaced based on the map (`variableMap`).  I've implemented a regex there and I don't compare any values (old - new).  If we would like to do it like you mentioned then we will need to check if we change sth in any variable in the map. I am not sure if we need to complicate it 😅